docs/populate.md

```diff
@@ -878,3 +879,50 @@ userSchema.pre('find', function (next) {
 });
 ```
 Alternatively, you can check out the [mongoose-autopopulate plugin](http://npmjs.com/package/mongoose-autopopulate).
+
+<h3 id="transform"><a href="#transform">Manipulate populated objects</a></h3>
+
+We can manipulate populated objects using `transform`. This option lets you register a function with two arguments: the populated document, and the original id, used to populate the document. The function will be called on every populated document and will give us more control over the result of the `populate()` execution.
```

Comment:
Minor nitpick: we prefer to use 2nd person "You can manipulate populated objects" versus "We can manipulate populated objects". Also, we prefer to have 1 line per sentence. Minor issues, overall this is great.